docs/api/platform/flow_control.md

```diff
@@ -2,51 +2,68 @@
 
 We can use Blinky to explore flow control and task management in Arm Mbed OS applications. We'll look at automated actions first, then move on to handling user actions.
 
-## Flow control for automated actions
+## Flow control for delayed actions
```

Comment:
This changed from "automated" to "delayed" but the other one is still "manual". So now the contrast isn't as clear to me. 

Also, line 3 uses "automated" and "user", which doesn't match either header.

Reply:
Good question. I changed "automated" because it sounded too vague/wide as the section is just above delays.

Personally I think "Automating actions with delays" and "Handling user actions" are clearer titles - I'll also update the intro in line 3 for consistency.